webomat.at.website.json

```diff
@@ -0,0 +1,23 @@
+{
+   "providerId": "Webomat.at",
+   "providerName": "Webomat Test",
+   "serviceId": "website",
```

Comment:
From serviceName I assume this is a test env. I would use also accordingly serviceId like "service-test".